db/migrate/20231025055928_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.references :author, foreign_key: { to_table: :users }, index: true
+      t.references :post, foreign_key: true, index: true
+      t.timestamps
+    end
```

Comment:
- [ ] As required for the project and according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) given to you, kindly ensure that the `comments` have the same columns like in the ERD diagram. I noticed that you made confusion between table names. Make sure you fix that